app/javascript/elm/tests/MainTests.elm

```diff
@@ -984,4 +992,11 @@ updateTests =
                     |> update (UpdateHeatMapMaximum int)
                     |> Tuple.first
                     |> Expect.equal expected
+        , fuzz bool "ToggleIsSearchOn changes the state of model.isSearchOn" <|
```

Comment:
Have you considered not testing at all? Also, I see that you haven't tested `update MapMoved`. Any reasons for that?

Reply:
It just feels wrong - not testing at all. I know we discussed overtesting, but I feel uncomfortable not adding any tests when I add new logic. Probably because a few weeks ago when we were adding staff in angular you always suggested that I should do more testing 😄

I understand why it seems like the tests are unnecessary right now. But do you remember how it was joining aircasitng when frontend didn't have any testing at all? 
And if we don't add this test cases now, then no one is gonna add them any time soon, so if the project will grow in a few years it might get to a similar situation as before?

TLDR - I'm experimenting and trying to find a balance 😉

Reply:
I think the Angular case is a totally different one (Ruby and Rails too for that matter). TLDR: no type system and no managed runtime like in Elm.

I'm fine with having different opinions on this. As long as we can back them up with valid arguments ;) Let's just test mindfully and not by trying to cover everything possible